rdsbroker/broker.go

```diff
@@ -387,20 +391,79 @@ func (b *RDSBroker) LastOperation(instanceID string) (brokerapi.LastOperationRes
 	return lastOperationResponse, nil
 }
 
+func (b *RDSBroker) CheckAndRotateCredentials() error {
+	dbInstanceDetailsList, err := b.dbInstance.DescribeByTag("Broker Name", b.brokerName)
+	if err != nil {
+		b.logger.Error("Could not retrieve the list of DB instances", err)
```

Comment:
That doesn't feel right to me. I think the consumer of the function (either `main()` or the test suite) should know and report errors.

I agree that it can be useful to have additional context, but I suspect the kind of errors we'll get back from this function will give us enough hints about where they've come from.